backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -56,28 +54,15 @@ trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLoggin
   /**
     * Restart or resume a previously-started job.
     */
-  def recover: Future[BackendJobExecutionResponse]
+  def recover: Future[BackendJobExecutionResponse] = execute
```

Comment:
Nice. I said in standup today that I was going to add a ticket for exactly this and you beat me to it